msteams-platform/messaging-extensions/api-based-overview.md

```diff
@@ -42,373 +40,18 @@ Before you get started, ensure that you adhere to the following requirements:
 
```

Comment:
For the two links above:
> * [OpenAPI Description (OAD)](#openapi-description)
> * [Update app manifest](#update-app-manifest)

Can you double check them as they don't render correctly for me?